extensions/amp-story/1.0/animation.js

```diff
@@ -60,7 +64,7 @@ const DEFAULT_EASING = 'cubic-bezier(0.4, 0.0, 0.2, 1)';
  * TODO(alanorozco): maybe memoize?
  */
 export function hasAnimations(element) {
-  const selector = `${ANIMATABLE_ELEMENTS_SELECTOR},amp-story-animation`;
+  const selector = `${ANIMATABLE_ELEMENTS_SELECTOR},>amp-story-animation`;
```

Comment:
TIL that `scopedQuerySelector` accepts comma-separated selectors. This is cool.

Reply:
It's been able for a little over a year now, I think :)